TUM-Dev/NavigaTUM
# Postgis as a postgres replacement

Moves us closer to the OSM goal

## Proposed Changes (include Screenshots if possible)

- Postgres -> PostGIS (a distribution of postgres)

## How to test this PR

1. launch the system
2. see that it still works

## How has this been tested?

- Letting it launch, import OSM data

## Checklist
